lib/iris/io/__init__.py

```diff
@@ -148,12 +148,12 @@ def expand_filespecs(file_specs):
         File paths which may contain '~' elements or wildcards.
 
     Returns:
-        A list of matching file paths.  If any of the file-specs matches no
+        A list of matching absolute file paths.  If any of the file-specs matches no
```

Comment:
When we have completed this PR, I think we should look at making the sort order of the returned filenames consistent and expected. The behaviour I think we need:

```
def expand_filespecs(file_specs):
    return list_of_expanded_filenames_sorted_by_filespecs_input_order
```